dqn.py

```diff
@@ -99,6 +115,9 @@ def compute_q_vals(self, states, actions):
 
         return result
 
+    def clear_max_q_val(self):
+        self.max_q_val = torch.tensor(0.0).to(self.device)
```

Comment:
Is there any reason why we might need something more than just a simple float?

Reply:
Perhaps so that when you take the max, there is no data transfer between CPU and GPU in this line:
```
        self.max_q_val = torch.max(self.max_q_val, result.abs().max())
```
I just like to keep all tensors on the same device.

I guess it absolutely doesn't matter for our task?